package.json

```diff
@@ -6,11 +6,14 @@
   "scripts": {
     "init": "npm ci && strapi build && node ./init/setup.js",
     "init:db": "node ./init/cleanup.js && node ./init/setup.js",
+    "init:test": "npm run init:db & node -e \"setTimeout(()=>{}, 20000)\" && strapi develop --no-build",
```

Comment:
The timeout is to make sure the tests don't hang?

Reply:
I might need to play with this more. Effectively, in init:db (setup.js) we run the whole process asynchronously and if I just call "npm run init:db && strapi develop", strapi starts without waiting for previous command to exit. One of the reason why I have refactored init scripts and extracted common library, to be able to call all those setup methods in a separate script.

Reply:
Updated to do initialization inside global test setup and teardown